packages/cli/src/cmds/init/initHandler.ts

```diff
@@ -67,12 +68,19 @@ export async function initHandler(argv: YargsArgs) {
 
   // Write main .js entry file
   const { src } = newArgs;
+
   try {
     if (pathUtils.basename(src) !== src) {
       await mkdirp(pathUtils.dirname(src));
     }
 
-    await fs.writeFile(src, template.source);
+    await fs.writeFile(
+      src,
+      isTemplateTypescript(argv.template)
```

Comment:
Could just store this as a variable at the start instead of using it multiple times.

`const isTypeScript = isTemplateTypescript(argv.template);`

Then use `isTypeScript` for your conditionals

Reply:
Yes, done that in the next commit.